### PR TITLE
Chapter1/homework/3

### PR DESCRIPTION
**Pull request by JereTR**
here's where I'm at, and doing a PR as requested.

### CODE REVIEW

**Comment by JereTR** [comment]
Sorry there's so many pulls, I kept wrapping my mind around this, and didn't want to lose a potential answer I was overlooking

File: Src/BootCamp.Chapter/Checks.cs [hunk 1]
```diff
@@ -16,26 +16,26 @@ public static class Checks
     {
         public static int PromptInt(string message)
         {
-            // To do: call your implementation. 
-            return 0;
+            //Lesson3.PromptInt(message);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Checks.cs:19]: Don't leave dead code. BURN IT WITH FIRE :)!

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:9]: Blank is not needed.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        public static void Demo()
+        {
+            Console.WriteLine("Demo works");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:12]: Demo itself should call the functions.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        }
+
+        public static float CalculateBmi(float weight, float height)
+        {
+            var Bmi = weight / height / height;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:17]: local variables start with non capital letter (camel Case)

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 3 above)
Comment (Almantask) [reply Src/BootCamp.Chapter/Lesson3.cs:12]: I think that is the intention, you just need some time to complete ;)

Comment (JereTR) [reply Src/BootCamp.Chapter/Lesson3.cs:12]: I am unsure how to call a function within a function.  The tests says it cannot find the functions if I put them within demo()

File: Src/BootCamp.Chapter/Checks.cs (hunk 1 above)
Comment (JereTR) [reply Src/BootCamp.Chapter/Checks.cs:19]: Done

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 2 above)
Comment (JereTR) [reply Src/BootCamp.Chapter/Lesson3.cs:9]: Didn't think of that, easy to remove.

File: Src/BootCamp.Chapter/Lesson3.cs (hunk 4 above)
Comment (JereTR) [reply Src/BootCamp.Chapter/Lesson3.cs:17]: My bad.  Easy fix.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+         //not sure how to call functions within here.  Test says it cannot find them.
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs:11]: "Test says it cannot find them"- Demo method is not tested, it's just for you, to run from the console and see if it works. I presume that's what you meant, if not- let me know.

To run a function from a function is quite simple- same rules apply (and you actually already do it)
```
// person 1
var age = PromptInt("Please enter age");
...
// person 2
...
```
person1 and person2 processing should ideally be a function as well. Same function called twice.

Let me know if it's clear how to proceed. :)

Comment (JereTR) [reply Src/BootCamp.Chapter/Lesson3.cs]: Ok.  Took a couple days and came back to this with a fresh mind.  I think this is what you're looking for.

I think I was spending too much time on just trying to figure this out, that my mind just wasn't clicking after a few hours.

**Pull request closed without merging**